Formula/wapm.rb

```diff
@@ -0,0 +1,35 @@
+class Wapm < Formula
+  desc "📦 WebAssembly Package Manager (CLI)"
```

Comment:
I'd rather not have non-ascii characters in the formula description. Folks can use environment variables to disable homebrew's utf8 messages. There will be no way to disable this on terminals that may not be compatible.

Reply:
That's fine. I didn't initially include it, but I saw such a char in the `wasmer` formula too, so... Would you like me to remove that in this PR too? Otherwise you can just make a simple commit I guess.

Reply:
I'd say let's leave `wasmer` alone for now, but remove it from this formula.